requirements.txt

```diff
@@ -1,6 +1,6 @@
 astroid==2.0.0
 botocore 
-cryptography 
+cryptography=2.4.2
```

Comment:
Does it need to be exactly this or would `>=` work?

Reply:
It does because of a issue with cryptography 5.0.0: https://github.com/paramiko/paramiko/issues/1369

Reply:
I will do a bit more playing around and see if I can still get it working with >= 2.4.2 though